HuntTheWumpus/HuntTheWumpus/Map.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HuntTheWumpus
+{
+    public static class Map
+    {
+        private static Random _random = new Random();
+        public static List<Room> Rooms { get; set; } = new List<Room>(21);
+
+        public static void PopulateRooms()
+        {
+            Rooms.Add(new Room() { ConnectedRooms = new List<int> { 0, 0, 0 } }); // Room 0
```

Comment:
Suggestion: Room 0 isn't needed as a staging area. When the game starts, the player should just be placed in a random room without a hazard.